src/pyedb/dotnet/edb_core/components.py

```diff
@@ -871,7 +871,6 @@ def create_port_on_pins(self, refdes, pins, reference_pins, impedance=50.0, port
                 elif "-" in ref_pin_name and ref_pin_name.split("-")[1] in refdes_pins:
                     ref_cmp_pins.append(refdes_pins[ref_pin_name.split("-")[1]])
             if not ref_cmp_pins:
-                self._logger.warning("No reference pins found during port creation. Port is not defined.")
```

Comment:
Why do you remove this warning ? Can you make it a debug warning if you don't want people to see it ?